chain/network/Cargo.toml

```diff
@@ -20,9 +20,12 @@ strum = { version = "0.20", features = ["derive"] }
 near-rust-allocator-proxy = "0.2.9"
 bytesize = "1.0.1"
 conqueue = "0.4.0"
+bs58 = "0.4.0"
 
 borsh = "0.9"
 cached = "0.23"
+serde = { version = "1", features = [ "derive" ] }
+serde_json = "1.0.55"
```

Comment:
I feel that `serde` and `serde_json` should not be used in near-network crate since our p2p protocol only uses borsh. If serde/JSON dependencies are required only for jsonrpc usage, I would consider creating counterpart types on jsonrpc-primitives crate (under `adversarial feature-flag)

Reply:
That makes sense. I'll create `near-network-primitives` and put structures with `Serialize` / `Deserialize` there.

Reply:
Removed